testing/vcs/test_box_custom_as_def_vistrails_exts.py

```diff
@@ -0,0 +1,29 @@
+import vcs
+import cdms2
+import os
+import sys
+pth = os.path.join(os.path.dirname(__file__),"..")
+sys.path.append(pth)
+import checkimage
+
+f=cdms2.open(os.path.join(vcs.prefix,"sample_data","clt.nc"))
+s=f("clt",slice(0,1),squeeze=1)
+
+x=vcs.init()
+x.drawlogooff()
+x.setbgoutputdimensions(1200,1091,units="pixels")
+
+gm=x.createboxfill()
+gm.boxfill_type="custom"
+gm.levels=[1.e20,1.e20]
+gm.ext_1="y"
+gm.ext_2="y"
+
+x.plot(s,gm,bg=1)
+fnm = "test_box_custom_as_def_vistrails_exts.png"
+src =sys.argv[1]
+x.png(fnm)
+print "fnm:",fnm
+print "src:",src
+ret = checkimage.check_result_image(fnm+'.png',src,checkimage.defaultThreshold)
```

Comment:
Any luck getting something together I can build?

We are getting pretty short on time.

Thanks,
Jerry


On Apr 20, 2015, at 11:03 AM, Charles Doutriaux <notifications@github.com<mailto:notifications@github.com>> wrote:


In testing/vcs/test_box_custom_as_def_vistrails_exts.py<https://github.com/UV-CDAT/uvcdat/pull/1221#discussion_r28712962>:

> +x.drawlogooff()
> +x.setbgoutputdimensions(1200,1091,units="pixels")
> +
> +gm=x.createboxfill()
> +gm.boxfill_type="custom"
> +gm.levels=[1.e20,1.e20]
> +gm.ext_1="y"
> +gm.ext_2="y"
> +
> +x.plot(s,gm,bg=1)
> +fnm = "test_box_custom_as_def_vistrails_exts.png"
> +src =sys.argv[1]
> +x.png(fnm)
> +print "fnm:",fnm
> +print "src:",src
> +ret = checkimage.check_result_image(fnm+'.png',src,checkimage.defaultThreshold)


I did, next time I will send the link to dashboard.

—
Reply to this email directly or view it on GitHub<https://github.com/UV-CDAT/uvcdat/pull/1221/files#r28712962>.

--
Gerald (Jerry) Potter
NASA Goddard Space Flight Center
E-mail  gerald.potter@nasa.gov<mailto:gerald.potter@nasa.gov>
Office  28:S241
Phone    707.315.8314
Analysis and Data Consultant
Department      GSFC:606.2